concent_api/core/tests/test_unit_validation.py

```diff
@@ -233,3 +235,36 @@ def test_that_if_different_public_keys_are_given_method_should_return_false(self
         subtask_results_accepted_list = self.create_subtask_results_accepted_list(self.task_to_compute_1, self.task_to_compute_2)
         result = are_keys_and_addresses_unique_in_message_subtask_results_accepted(subtask_results_accepted_list)
         self.assertFalse(result)
+
+
+class TestFramesListValidaation(TestCase):
+
+    def test_that_list_of_ints_is_valid(self):
+        try:
+            validate_frames([1, 2])
+        except Exception:  # pylint: disable=broad-except
+            self.fail()
```

Comment:
Please remove these exception handlers. Pylint is right here.

That's exactly the reason why we have https://github.com/Code-Poets/coding-style/issues/8
All the warnings you're silencing should be listed in a comment for review.